### PR TITLE
feat: add autobump flow

### PR DESCRIPTION
**Pull request by chenrui333**
Similar workflow as we used in core and cask, set check frequency to daily

relates to #203

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]
Looks good, thanks @chenrui333 and @SMillerDev!

**Review by SMillerDev — APPROVED** [review]

File: .github/workflows/autobump.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          test-bot: false
+
+      - name: Configure Git user
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: ${{ (github.event_name == 'workflow_dispatch' && github.actor) || 'BrewTestBot' }}
+
+      - name: Get list of autobump portable-ruby formulae
+        id: autobump
+        run: echo "autobump_list=$(xargs < "$(brew --repo homebrew/homebrew-portable-ruby)"/.github/autobump.txt)" >> "$GITHUB_OUTPUT"
```
Comment (Bo98) [inline .github/workflows/autobump.yml:46]: Rather not have to duplicate names into an autobump.txt:

```suggestion
        run: echo "autobump_list=$(brew tap-info --json "${GITHUB_REPOSITORY}" | jq -r '.[].formula_names[]')" >> "$GITHUB_OUTPUT"
```

File: .github/workflows/autobump.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+env:
+  HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+
+permissions:
+  contents: read
+
+jobs:
+  autobump:
+    if: github.repository == 'Homebrew/homebrew-portable-ruby'
```
Comment (Bo98) [inline .github/workflows/autobump.yml:26]:
```suggestion
    if: github.repository_owner == 'Homebrew'
```

Minor but more concise

Comment (chenrui333) [reply .github/workflows/autobump.yml]: yeah, works for me 👍

File: .github/workflows/autobump.yml (hunk 1 above)
Comment (chenrui333) [reply .github/workflows/autobump.yml]: adopted 👍

File: .github/autobump.txt
Comment (Bo98) [inline .github/autobump.txt:1]: Should be able to remove this file now

**Review by MikeMcQuaid — APPROVED** [review]
Looks good, thanks again @chenrui333 and everyone else for the help!

**Comment by chenrui333** [comment]
need a rerun for the ci? not sure why it failed

**Pull request merged**